apps/Standalone/src/desginer/state/workflowLoadingSlice.ts

```diff
@@ -27,6 +27,7 @@ export interface WorkflowLoadingState {
   hostOptions: {
     displayRuntimeInfo: boolean; // show info about where the action is run(i.e. InApp/Shared/Custom)
     forceEnableSplitOn?: boolean; // force enable split on for all actions
+    useDisplaytextEditor?: boolean; // show displaytext editor
```

Comment:
Instead of choosing to use vs not use the editor, can we make this host specific to DM connector? The token editors are supposed to be extendable to the connector owners, and if we're limiting it on the host-level, then we could have just used the custom editor.